packages/zoe/src/contracts/simpleExchange.js

```diff
@@ -22,57 +27,39 @@ import '../../exported';
  * not partially fill orders.
  *
  * The invitation returned on installation of the contract is the same as what
- * is returned by calling `await E(publicAPI).makeInvite().
+ * is returned by calling `await E(publicAPI).makeInvitation().
```

Comment:
Installation of a contract does not return an invitation. What is the significance of the `await` above? An `await` is not something one calls.